utils/format.ts

```diff
@@ -37,7 +37,7 @@ export function capitalize(s: string): string {
 }
 
 export function formatWeiString(v: ethers.BigNumberish): number {
-  return Number(ethers.utils.formatEther(v));
+  return v ? Number(ethers.utils.formatEther(v)) : 0;
```

Comment:
this is strange to me since typescript should be checking this, perhaps theres some data validation issue outside of this?

Reply:
I discovered the problem. One LSP contract is missing from the Tvl table, so the field is missing from the api. I made the field optional in the model